src/containers/Accounts/AccountAssetTab/AccountAssetTab.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ChangeEvent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { useHistory, useParams } from 'react-router';
+import { AccountTokenTable } from '../AccountTokenTable';
+import { AccountNFTTable } from '../AccountNFTTable/AccountNFTTable';
+
+// TODO: Add state types or convert to react query
+interface Props {
+  account: any;
+}
+
+const AccountAssetTabDisconnected = ({ account }: Props) => {
+  const assetTypes = ['token', 'nft'];
+  const { id: accountId, assetType = assetTypes[0] } = useParams<{
+    id: string;
+    assetType: string;
+  }>();
+  const supportsNFT = process.env.REACT_APP_ENVIRONMENT === 'nft_sandbox';
```

Comment:
`devnet` also supports NFTs currently :)

Reply:
I will need to update this.